ibis/backends/dask/__init__.py

```diff
@@ -0,0 +1,98 @@
+from __future__ import absolute_import
+
+import toolz
+
+import ibis.config
+from ibis.backends.base_sqlalchemy.compiler import Dialect
+
+from .client import DaskClient
+from .execution import execute, execute_node
+from .udf import udf
+
+__all__ = ('connect', 'dialect', 'execute', 'udf')
+
+
+with ibis.config.config_prefix('dask'):
+    ibis.config.register_option(
+        'enable_trace',
+        False,
+        'Whether enable tracing for dask execution. '
+        'See ibis.dask.trace for details.',
+        validator=ibis.config.is_bool,
+    )
+
+
+def connect(dictionary):
+    """Construct a dask client from a dictionary of DataFrames.
+
+    Parameters
+    ----------
+    dictionary : dict
+
+    Returns
+    -------
+    DaskClient
+    """
+    return DaskClient(dictionary)
+
+
+def from_dataframe(df, name='df', client=None):
+    """
+    convenience function to construct an ibis table
+    from a DataFrame
+
+    EXPERIMENTAL API
+
+    Parameters
+    ----------
+    df : DataFrame
+    name : str, default 'df'
+    client : Client, default new DaskClient
+        client dictionary will be mutated with the
+        name of the DataFrame
+
+    Returns
+    -------
+    Table
+    """
+
+    if client is None:
+        return connect({name: df}).table(name)
+    client.dictionary[name] = df
+    return client.table(name)
+
+
+def _flatten_subclass_tree(cls):
```

Comment:
ideally put things like this else where *maybe* dask/expr.py or similiar

Reply:
Should the pandas backend functions be moved to a `pandas/expr.py` as well? Or is there a good place to start putting functions that are directly shared between dask and pandas? I could import this directly from the pandas backend actually.

Reply:
so yes and yes. however these should be done independently. (e.g. separate PRs for pandas). may make sense to move to a common location (or dask can simply import the pandas ones)